test/controllers/foods_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class FoodsControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get foods_index_url
+    assert_response :success
+  end
+
+  test "should get show" do
+    get foods_show_url
+    assert_response :success
+  end
+
+end
```

Comment:
In addition to the feature test you have for this user story, you'll want to test any other classes you created. You have `FoodSearch` and `FoodServices`, and those should be tested also.